src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import { FC } from 'react';
+import { Todo } from '../../api/todos';
+import usersFromServer from '../../api/users';
+import { User } from '../../api/users';
+import { UserInfo } from '../UserInfo';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: FC<Props> = props => {
+  const { todo } = props;
+  const { id, completed, title, userId } = todo;
+
+  const filteredUser = usersFromServer.find(
+    (user: User): boolean => user.id === userId,
+  );
```

Comment:
The `filteredUser` might be `undefined` if no user with the matching `userId` is found. Consider handling this case to avoid potential errors when rendering the `UserInfo` component.

Reply:
I do check in  other function